loadtest/user/userentity/actions.go

```diff
@@ -135,6 +135,22 @@ func (ue *UserEntity) UpdateUser(user *model.User) error {
 	return nil
 }
 
+func (ue *UserEntity) UpdateUserRoles(userId, roles string) error {
+	ok, err := ue.IsSysAdmin()
+	if err != nil {
+		return err
+	}
+	if !ok {
+		return nil
+	}
```

Comment:
I think we shouldn't return `nil` here, not being able to update user should result an error. In the ` cmd/ltagent/init.go:L91` we check the `error` to consider action is successful.